test/zdtm.py

```diff
@@ -1329,6 +1344,9 @@ def dump(self, action, opts=[]):
         os.mkdir(self.__ddir())
         os.chmod(self.__ddir(), 0o777)
 
+        if self.__rootless:
+            os.setgid(NON_ROOT_UID)
```

Comment:
I don't think that we need to change uid in zdtm.py. I think it should be better if a caller executes zdtm.py from a non-root user.

Reply:
Removed in 2 placed, thanks.